pages/common/wkhtmltopdf.md

```diff
@@ -0,0 +1,24 @@
+# wkhtmltopdf
+
+> An open-source command-line tool to convert HTML documents or web pages into PDF files.
+> More information: <https://wkhtmltopdf.org/>.
+
+- Basic Conversion:
+
+`wkhtmltopdf {{input.html}} {{output.pdf}}`
+
+- Specify Page Size (please see `PaperSize` of `QPrinter` for supported sizes):
+
+`wkhtmltopdf --page-size {{A4}} {{input.html}} {{output.pdf}}`
+
+- Set Margins:
+
+`wkhtmltopdf --margin-{{top|bottom|left|right}} {{10mm}} {{input.html}} {{output.pdf}}`
+
+- Custom Page Orientation:
+
+`wkhtmltopdf --orientation {{Landscape|Portrait}} {{input.html}} {{output.pdf}}`
+
+- Generate Grayscale PDF:
```

Comment:
```suggestion
- Generate a greyscale version of the PDF document:
```

Note that as of now [our style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) doesn't specify which version of English (American or British) these pages should be written in.

Reply:
Done @reinhart1010